.circleci/config.yml

```diff
@@ -512,6 +512,29 @@ jobs:
           command: make && git diff --exit-code
           working_directory: op-bindings
 
+  bindgen-remote:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run:
+          name: bindgen remote bindings
+          command: make bindgen-remote && git diff --exit-code
+          working_directory: op-bindings
+
+  bindgen-test:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run:
+          name: bindgen test
+          command: ETHERSCAN_APIKEY_ETH=foo ETHERSCAN_APIKEY_OP=foo RPC_URL_ETH=foo RPC_URL_OP=foo make test-bindgen
```

Comment:
are these env placeholders required?

Reply:
Sorry meant to keep this as a draft PR until I figure out how to configure the Circle CI context to fill these ENVs in